maquetado/scss/tech.scss

```diff
@@ -1,58 +1,96 @@
-@import 'colors';
-@import 'fonts';
-@import 'sizes';
+@import "colors";
+@import "fonts";
+@import "sizes";
 @mixin card-size {
-    width: 200px;
+  width: 200px;
 }
 
 .main-container {
-    padding: 100px 50px;
-    z-index: 1;
+  padding: 100px 50px;
+  z-index: 1;
 }
 
 .tc-container {
-    display: grid;
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: repeat(2, 2fr);
-    width: max-content;
-    margin: auto;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 2fr);
+  width: max-content;
+  margin: auto;
 }
 
 .left-card {
-    grid-row: 1/3;
-    align-self: center;
-    border-right: $green solid 2px;
-    height: 600px;
-    padding: 150px 30px;
+  grid-row: 1/3;
+  align-self: center;
+  border-right: $green solid 2px;
+  height: 600px;
+  padding: 150px 30px;
 }
 
-.tech-name, .tech-release, .tech-info { margin: 5px; }
+.tech-name,
+.tech-release,
+.tech-info {
+  margin: 5px;
+}
 
 .tech-card {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    margin: 40px 20px;
-    position: relative;
-    @include card-size();
-
-    .tech-logo {
-        width: $tech-logo-size;
-        height: $tech-logo-size;
-        margin-bottom: 10px;
-    }
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 40px 20px;
+  position: relative;
+  @include card-size();
 
-    .tech-name {
-        font-size: $text-medium;
-    }
+  .tech-logo {
+    width: $tech-logo-size;
+    height: $tech-logo-size;
+    margin-bottom: 10px;
+  }
 
-    .tech-release {
-        font-weight: bold;
-    }
+  .tech-name {
+    font-size: $text-medium;
+  }
 
-    .tech-info {
-        text-align: center;
-        font-size: $text-small;
-    }
+  .tech-release {
+    font-weight: bold;
+  }
+
+  .tech-info {
+    text-align: center;
+    font-size: $text-small;
+  }
 }
 
+@media (max-width: 1024px) {
+  .tc-container {
+    grid-template-rows: repeat(5, 1fr);
```

Comment:
You can define the amount of rows as `auto-fill`, so you dont have to worry about the amount of techs and if there are more in the future. 
Doing this, you can also avoid defining a different class for the back-end techs sections.